lib/mbta_v3_api/stops/repo.ex

```diff
@@ -103,7 +103,7 @@ defmodule MBTAV3API.Stops.Repo do
       fn stop ->
         stop
         |> by_route_type_fn.()
-        |> Enum.map(&get_parent/1)
+        #removed "get parent call" to reduce number of API calls when getting stops
```

Comment:
```suggestion
        #removed "get_parent call" to reduce number of API calls when getting stops
```
tiny suggestion to make this searchable with `get_parent`, when someone is trying to figure out the usage